model/task_lifecycle.go

```diff
@@ -925,6 +925,19 @@ func UpdateBuildStatus(b *build.Build) (bool, error) {
 		return false, nil
 	}
 
+	isAborted := false
```

Comment:
Can we add a comment that's like "only need to check if aborted has changed if status has changed" or something so it's clear that this is intentional?